src/main/java/gil/server/controllers/PostPersonController.java

```diff
@@ -0,0 +1,49 @@
+package gil.server.controllers;
+
+import gil.server.data.FileIO;
+import gil.server.data.JSONDataStore;
+import gil.server.data.Person;
+import gil.server.http.Request;
+import gil.server.http.Response;
+
+import javax.json.JsonObject;
+import java.io.IOException;
+import java.util.function.BiFunction;
+
+public class PostPersonController {
```

Comment:
usually, you'd have one controller for one resource and have multiple functions within it for different methods

Reply:
That makes sense. I refactored the PersonController class to include the post lambda function.